docs/integrations/sources/facebook-marketing.md

```diff
@@ -6,87 +6,94 @@ This page guides you through the process of setting up the Facebook Marketing so
 <!-- /env:oss -->
 
 ## Setup guide
 
-<!-- env:cloud -->
+<!-- env:oss -->
+
+### (For Airbyte Open Source) Generate an access token and request a rate limit increase:
 
-### For Airbyte Cloud:
+To set up Facebook Marketing as a source in Airbyte Open Source, you will first need to create a Facebook app and generate a Marketing API access token. You will then need to request a rate limit increase from Facebook. The following steps will guide you through this process:
+
+1. Navigate to [Meta for Developers](https://developers.facebook.com/apps/) and follow the steps provided in the [Facebook documentation](https://developers.facebook.com/docs/development/create-an-app/) to create a Facebook app. Set the app type to **Business** when prompted.
+2. From your App’s dashboard, [set up the Marketing API](https://developers.facebook.com/docs/marketing-apis/get-started).
+3. Generate a Marketing API access token: From your App’s Dashboard, click **Marketing API** --> **Tools**. Select all the available token permissions (`ads_management`, `ads_read`, `read_insights`, `business_management`) and click **Get token**. Copy the generated token for later use.
+4. Request a rate limit increase: Facebook [heavily throttles](https://developers.facebook.com/docs/marketing-api/overview/authorization#limits) API tokens generated from Facebook apps with the default Standard Access tier, making it infeasible to use the token for syncs with Airbyte. You'll need to request an upgrade to Advanced Access for your app on the following permissions:
 
-**To set up Facebook Marketing as a source in Airbyte Cloud:**
+   - Ads Management Standard Access
+   - ads_read
+   - Ads_management
 
-1. [Log into your Airbyte Cloud](https://cloud.airbyte.io/workspaces) account.
+   See the Facebook [documentation on Authorization](https://developers.facebook.com/docs/marketing-api/overview/authorization/#access-levels) to request Advanced Access to the relevant permissions.
+
+<!-- /env:oss -->
+
+### Set up Facebook Marketing as a source in Airbyte:
+
+1. [Log in to your Airbyte Cloud](https://cloud.airbyte.io/workspaces) account, or navigate to your Airbyte Open Source dashboard.
 2. In the left navigation bar, click **Sources**. In the top-right corner, click **+ New source**.
-3. On the Set up the source page, select **Facebook Marketing** from the **Source type** dropdown.
-4. For Name, enter a name for your Facebook Marketing connector.
+3. Find and select **Facebook Marketing** from the list of available sources.
+4. For **Source name**, enter a name for your Facebook Marketing connector.
+5. To authenticate the connection:
+
+   <!-- env:cloud -->
+   **For Airbyte Cloud**: Click **Authenticate your account** to authorize your [Meta for Developers](https://developers.facebook.com/) account. Airbyte will authenticate the account you are already logged in to. Make sure you are logged into the right account.
```

Comment:
why Meta For Developers account? do I need to be registered as a meta developer to use on Airbyte Cloud? That doesn't sound right

Reply:
Sorry bout that, got my Metas and Facebooks mixed up!